ansible-ipi-install/README.md

```diff
@@ -298,6 +298,7 @@ pullsecret=""
 # The hardware_profile is used by the baremetal operator to match the hardware discovered on the host
 # See https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#baremetalhost-status
 # ipmi_port is optional for each host. 623 is the common default used if omitted
+# poweroff=false indicates the playbook shouldn't power off the node before deploying OCP. True by default
 [masters]
 master-0 name=master-0 role=master ipmi_user=admin ipmi_password=password ipmi_address=192.168.1.1 ipmi_port=623 provision_mac=ec:f4:bb:da:0c:58 hardware_profile=default
```

Comment:
I'd place the option on all the systems like so
```
master-0 name=master-0 role=master ipmi_user=admin ipmi_password=password ipmi_address=192.168.1.1 ipmi_port=623 provision_mac=ec:f4:bb:da:0c:58 hardware_profile=default poweroff=true
```
This your code already assumes true its undefined, but at least they see it clearly in the README where it should be placed it they want to change it.

Reply:
Well... See https://github.com/openshift-kni/baremetal-deploy/pull/280#discussion_r407506878
At first I made it mandatory but after @sreichar 's comment I changed it to optional (which I prefer).
You decide, mandatory or optional.

Reply:
I think it should be optional BUT still show how to use it in the README. For example, ipmi_port is optional but we show the example of were to set it in the sample inventory. If someone omits it, no problem uses the default port which we set. In your case, we would just add poweroff=true. Currently as it stands, I don't think it is clear where the option should reside without an example.